app/assets/stylesheets/book.css.scss

```diff
@@ -255,19 +255,21 @@ sub.beta-subscript{
   color: white;
 }
 
-#tag-and-searchbar{
-  padding-top: 2rem;
+.bars-and-books {
+  position: absolute;
   width: 100%;
-  margin-left: auto;
-  margin-right: auto;
   margin-top: -108px;
 }
 
+#tag-and-searchbar {
+  padding: 2rem 0 2rem 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `2rem 0` instead of `2rem 0 2rem 0`